tests/unit/Codeception/Module/Db/DbCleaningTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+abstract class DbCleaningTest extends \PHPUnit_Framework_TestCase
```

Comment:
Wouldn't it be better to convert all these function specific files to traits and combine then to a single Test class per database type?